apps/src/codebridge/Codebridge.tsx

```diff
@@ -18,13 +16,15 @@ import {
 import classNames from 'classnames';
 import React, {useEffect, useMemo, useReducer, useRef} from 'react';
 
-import {FilePreview} from '@cdo/apps/codebridge/FilePreview';
+import {FileBrowser} from '@cdo/apps/codebridge/FileBrowser/FileBrowser';
```

Comment:
Just out of curiosity, is there a reason you switched from `@codebridge/FileBrowser' to '@cdo/apps/codebridge/FileBrowser`?

Reply:
autocomplete 🙃 I'll update